mLRS/Common/hal/esp/esp-hal.h

```diff
@@ -88,6 +88,10 @@
 #include "tx-hal-radiomaster-rp4td-2400-esp32.h"
 #endif
 
+#ifdef TX_ELRS_RADIOMASTER_TX16S_2400_ESP32
+#include "tx-hal-radiomaster-tx16s-2400-esp32.h"
```

Comment:
I guess I'd like some indication of int also in file name

Reply:
You also chose this name.  Does the "I guess" indicate I don't need to change it? :-)

Reply:
you don't have to, but I think we want to think about the naming ... for the define due to the ELRS in it I find it easier than for the hal name